egs/fame/README.txt

```diff
@@ -6,10 +6,10 @@ The Corpus consists of short utterances extracted from 203 audio segments of app
 
 A full description of the FAME! Speech Corpus is provided in:
 
-Yilmaz, E., Heuvel, H. van den, Van de Velde, H., Kampstra, F., Algra, J., Leeuwen, D. van:
+E. Yılmaz, H. van den Heuvel, J. Dijkstra, H. Van de Velde, F. Kampstra, J. Algra and D. van Leeuwen, "Open Source Speech and Language Resources for Frisian,"  In Proc. INTERSPEECH, pp. 1536-1540, San Francisco, CA, USA, Sept. 2016. 
 
-Open Source Speech and Language Resources for Frisian Language.
+Speaker clustering and verification corpus details are provided in:
```

Comment:
I didn't see anything about speaker clustering in the PR. Did I miss something?

Reply:
No, it is indeed the case. In the corpus, you will see that SC folder contains speaker diarized segments (in SC/SD) as well as raw segments (in SC/raw). The diarized segments are used for training the speaker verification system (UBM, ivector, plda training etc..) and used in local/prepare_train.sh.